src/frontend/src/lib/util/util.ts

```diff
@@ -66,17 +76,42 @@ export const mod_panel_permissions = [
 
 export const valid_games: { [key: string]: string } = {
   mk8dx: 'Mario Kart 8 Deluxe',
+  mk8: 'Mario Kart 8',
   mkw: 'Mario Kart Wii',
   mkt: 'Mario Kart Tour',
+  mk7: 'Mario Kart 7',
+  smk: 'Super Mario Kart',
 };
 export const valid_modes: { [key: string]: string[] } = {
-  mk8dx: ['150cc', '200cc'],
+  mk8dx: [
+    '150cc',
+    '200cc',
+    'mixed_battle',
+    'ballon_battle',
+    'shine_thief',
+    'bobomb_blast',
+    'coin_runners',
+    'renegade_roundup',
+    'match_race',
+    'mixed',
+  ],
+  mk8: ['150cc', '200cc'],
   mkw: ['rt', 'ct'],
   mkt: ['vsrace'],
+  mk7: ['vsrace'],
+  smk: ['match_race'],
 };
 export const mode_names: { [key: string]: string } = {
   '150cc': '150cc',
   '200cc': '200cc',
+  mixed_battle: 'Battle (Mixed)',
+  ballon_battle: 'Ballon Battle',
```

Comment:
minor nitpick but it's spelled "balloon battle" so probably want to fix the variable name and string